File: solutions/julia/raindrops/1/raindrops.jl
function raindrops(number)
	retval = ""
	divflag = false
	if number % 3 == 0
		divflag = true
		retval = string(retval, "Pling")
	end
	if number % 5 == 0
		divflag = true
		retval = string(retval, "Plang")
	end
	if number % 7 == 0
		divflag = true
		retval = string(retval, "Plong")
	end
	if divflag == false
		retval = string(number)
	end
	return retval
end

File: solutions/cpp/raindrops/1/raindrops.cpp
#include "raindrops.h"

namespace raindrops {
    std::string convert (int num) {
        std::string retVal;
        if (!(num % 3)) {
            retVal = "Pling";
        }
        if (!(num % 5)) {
            retVal = retVal + "Plang";
        }
        if (!(num % 7)) {
            retVal = retVal + "Plong";
        }
        if (!retVal.length()) {
            retVal = std::to_string(num);
        }
        return retVal;
    }
}  // namespace raindrops
